nuget/lib/dependabot/nuget/file_parser.rb

```diff
@@ -22,8 +24,36 @@ def parse
         dependencies
       end
 
+      sig { returns(Ecosystem) }
+      def ecosystem
+        @ecosystem ||= T.let(
+          Ecosystem.new(
+            name: ECOSYSTEM,
+            package_manager: package_manager,
+            language: language
+          ),
+          T.nilable(Ecosystem)
+        )
+      end
+
       private
 
+      sig { returns(T.nilable(T::Array[T.nilable(String)])) }
+      def content_json
+        @content_json ||= T.let(begin
+          directory = source&.directory || "/"
+          discovery_json_reader = DiscoveryJsonReader.run_discovery_in_directory(
+            repo_contents_path: T.must(repo_contents_path),
+            directory: directory,
+            credentials: credentials
+          )
+
+          discovery_json_reader.workspace_discovery&.projects&.map do |framework|
+            T.let(framework.instance_variable_get(:@target_frameworks), T::Array[String]).first
```

Comment:
.NET projects can target multiple frameworks, so you're potentially omitting anything but the first listed one here and your telemetry may not be giving an accurate picture.

Reply:
@JamieMagee , understood , i have merged the versions as a single string now.